src/main/java/uk/gov/indexer/ExceptionFormatter.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.indexer;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class ExceptionFormatter {
+    public static String formatExceptionAsString(Exception e) {
+        StringWriter sw = new StringWriter();
+        PrintWriter pw = new PrintWriter(sw);
+        e.printStackTrace(pw);
+        return sw.toString();
+    }
```

Comment:
Alternatives to this home-grown method include the [commons-lang.ExceptionUtils::getStackTrace() function](
https://commons.apache.org/proper/commons-lang/javadocs/api-3.4/org/apache/commons/lang3/exception/ExceptionUtils.html#getStackTrace%28java.lang.Throwable%29).